### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by bbredewold** [JohanDegraeve/xdripswift#504] [General]
Hi people!

As a daily user I just wanted to thank all of you developing this iOS App.
I love the fact that I can use my Libre 2's wirelessly and connect it to Nightscout and Apple Health.

I wish I could return the favor and help maintaining this repo in any form, but as a webdeveloper I'm lacking iOS development skills. So, if there are any other things I could help with, please let me know.

Cheers!

Bert

**Comment by paulplant** [comment]
Thank you!
